app/controllers/import_controller.rb

```diff
@@ -203,82 +194,55 @@ def validate_required_field(field, value, row_ind)
-    return NORMALIZED_ENUMS[field][normalized_value] if NORMALIZED_ENUMS[field].keys.include?(normalized_value)
-
-    err_msg = "'#{value}' is not an acceptable value for '#{VALIDATION[field][:label]}',"\
-              " acceptable values are: #{VALID_PATIENT_ENUMS[field].reject(&:blank?).to_sentence}"
-    raise ValidationError.new(err_msg, row_ind)
+    NORMALIZED_ENUMS[field].keys.include?(normalized_value) ? NORMALIZED_ENUMS[field][normalized_value] : value
   end
 
-  def validate_bool_field(field, value, row_ind)
+  def import_and_validate_bool_field(field, value, row_ind)
     return value if value.blank?
     return (value.to_s.downcase == 'true') if %w[true false].include?(value.to_s.downcase)
 
-    err_msg = "'#{value}' is not an acceptable value for '#{VALIDATION[field][:label]}', acceptable values are: 'True' and 'False'"
+    err_msg = "Value '#{value}' for '#{VALIDATION[field][:label]}' is not an acceptable value, acceptable values are: 'True' and 'False'"
     raise ValidationError.new(err_msg, row_ind)
   end
 
-  def validate_date_field(field, value, row_ind)
-    return nil if value.blank?
-    return value if value.instance_of?(Date)
-
-    unless value.match(/\d{4}-\d{2}-\d{2}/)
-      err_msg = "'#{value}' is not a valid date for '#{VALIDATION[field][:label]}'"
-      if value.match(%r{\d{2}/\d{2}/\d{4}})
-        raise ValidationError.new("#{err_msg} due to ambiguity between 'MM/DD/YYYY' and 'DD/MM/YYYY', please use the 'YYYY-MM-DD' format instead", row_ind)
-      end
-
-      raise ValidationError.new("#{err_msg}, please use the 'YYYY-MM-DD' format", row_ind)
-    end
-
-    begin
-      Date.parse(value)
-    rescue ArgumentError
-      raise ValidationError.new("'#{value}' is not a valid date for '#{VALIDATION[field][:label]}'", row_ind)
-    end
+  def import_date_field(value)
+    value.blank? ? nil : value
   end
 
-  def validate_phone_field(field, value, row_ind)
-    return nil if value.blank?
-
-    phone = Phonelib.parse(value, 'US')
-    return phone.full_e164 unless phone.national(false).nil? || phone.national(false).length != 10
-
-    raise ValidationError.new("'#{value}' is not a valid phone number for '#{VALIDATION[field][:label]}'", row_ind)
+  def import_phone_field(value)
+    e_164 = Phonelib.parse(value, 'US').full_e164
+    e_164.blank? ? value : e_164
   end
 
-  def validate_state_field(field, value, row_ind)
+  def import_and_validate_state_field(field, value, row_ind)
     return nil if value.blank?
     return normalize_and_get_state_name(value) if VALID_STATES.include?(normalize_and_get_state_name(value))
 
     normalized_state = STATE_ABBREVIATIONS[value.upcase.to_sym]
     return normalized_state if normalized_state
 
+    # NOTE: Currently only import allows abbreviated state names. If that changes and we begin allowing abbreviations
+    # via other controllers, it will probably make sense to move this error onto the Patient model
```

Comment:
To respond to this comment, the validation for `address_state` on the Patient model that is now being done also on import won't allow abbreviated values which is a problem since that's expected to be supported by import.

Reply:
In this case, I'm not sure it is a problem to only allow state names on the Patient model. As far as I can tell, we've only ever saved `address_state` (and other state fields) to the database as abbreviations (and the console session we did to investigate the validation supports this). So I think that is the validation that we want on the Patient model. Non-abbreviated values are still supported in import because on the lines above, they get normalized to be an abbreviated value. Does my thinking make sense, or am I missing something?

Reply:
My mistake, I said this totally backwards. The `address_state` field is saved as a full state name (e.g. 'Ohio'). On the model, we validate that a state field is one of these full state names. Abbreviations are still allowed in import, because we normalize abbreviated values to full state names before validating.

Reply:
Nope, you're right. Was worried for a second the `import_controller` was not normalizing those abbreviated values, but we're good.